tree/ntuple/v7/test/ntuple_limits.cxx

```diff
@@ -201,20 +203,23 @@ TEST(RNTuple, DISABLED_Limits_ManyPagesOneEntry)
 
 TEST(RNTuple, DISABLED_Limits_LargePage)
 {
-   // Writing and reading one page with 100M elements takes around 3.5s and seems to have linear complexity (200M
-   // elements take 7s, 400M elements take 13.5s).
+   // Writing and reading one page with 600M elements takes around 18s and seems to have linear complexity
```

Comment:
Not sure if these specific numbers make sense to keep in the source, as I imagine they depend a lot on the underlying machine, build type etc. Maybe it's best to speak about their ratios or in more qualitative terms

Reply:
Yes, absolute numbers are expected to change, but I originally put them so that dramatic changes, for example because of runtime complexity, can be caught...